app/models/property.rb

```diff
@@ -0,0 +1,8 @@
+class Property < ApplicationRecord
+  belongs_to :assessment
+
+  validates :value, numericality: { greater_than: 0 }
+  validates :outstanding_mortgage, numericality: { greater_than_or_equal_to: 0 }
+  validates :percentage_owned, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 100 }
+  validates :main_home, :shared_with_housing_assoc, inclusion: { in: [true, false] }
```

Comment:
Aren't these validations duplicates of what is already checked by the JSON schema?